pages/client-libraries/python.mdx

```diff
@@ -3,86 +3,665 @@ title: Python quick start
+
+    # Get the result
+    for record in records:
+        print(record["num_of_nodes"])
+```
+
+Notice that the first argument in auth is the username and the second is the password.
+
+
+#### Connect with self-signed certificate
+
+By adding `bolt+ssc` before the host address in the URI argument, the connection will be encrypted with a self-signed certificate. You will need to use the certificate if you want to **connect to Memgraph Cloud** instance.
+
+When creating a project on Memgraph Cloud, you have a username and you create a password. After the project is started, Memgraph has assigned host address.
+Copy those values and paste them in the `<MEMGRAPH_HOST_ADDRESS>`, `<MEMGRAPH_USERNAME>` and `<MEMGRAPH_PASSWORD>` fields in the code snippet below to connect successfully to the database.
+
+```python
+from neo4j import GraphDatabase
+
+URI = "bolt+ssc://<MEMGRAPH_HOST_ADDRESS>"
+AUTH = (<MEMGRAPH_USERNAME>, <MEMGRAPH_PASSWORD>)
+
+
+with GraphDatabase.driver(URI, auth=AUTH) as driver:
+    driver.verify_connectivity()
+
+    # Count the number of nodes in the database
+    records, summary, keys = driver.execute_query(
+        "CREATE (u:User {name: $name, password: $password}) RETURN u.name AS name",
+        name="John",
+        password="pass",
+        database_="memgraph",
+    )
+
+    # Get the result
+    for record in records:
+        print(record["name"])
+```
+
+### Query the database 
+
+After connecting your driver to Memgraph, you can start running queries. The simplest way to run queries is by using the `execute_query()` method which has an automatic transaction management. 
+
+#### Run a create query 
+
+The following query will create a node inside the database: 
+
+```python
+node = "CREATE (n:Technology {name:'Memgraph'});"
+driver.execute_query(node)
+```
+
+Due to the nature of the `execute_query()` method, transactions are handled automatically. 
+
+#### Run a read query 
+
+The following query will read the previously created node from the database: 
+
+```python
+node = "MATCH (n:Technology) RETURN n;"
+result = driver.execute_query(node)
+
+for record in records:
+    node = record["n"] 
+    print(node)   
+
+```
+
+In this query, each record contains a node object behind the variable `n`.
+
+
+#### Running queries with property map
+
+```python
+query = "CREATE (n:Technology {id:$id, name: $name, description: $description});"
+driver.execute_query(query, 
+    id=1, 
+    name="MemgraphDB", 
+    description="Fastest graph database")
+```
+
+Using this approach, the queries will not contain hard-coded values, they can be more dynamic.
+
+### Proces the results 
+
+Processing results from the database is important since we do not want to lose any data during conversions and properly read results and serve them back to the Python application. Python is dynamically typed language, which means that the type of the variable is determined at runtime, which is why we need to be careful when processing results.
```

Comment:
```suggestion
Processing results from the database is important to avoid losing any data during conversions and properly read results and serve them back to the Python application. Python is a dynamically typed language, which means that the type of the variable is determined at runtime, which is why we need to be careful when processing results.
```